lib/Models/Terria.ts

```diff
@@ -1094,7 +1094,7 @@ export default class Terria {
       // a hanging promise if a valid viewer never becomes available,
       // for eg: when react is not rendered - `currentViewer` will always be `NoViewer`.
       await when(isViewerAvailable);
-      await this.currentViewer.zoomTo(firstMappableItem, 0.0);
+      await this.currentViewer.zoomTo(mappables, 0.0);
```

Comment:
Instead of zooming to first mappable in the workbench, we zoom to a view showing all the mappables.